include/sir/config.h

```diff
@@ -315,83 +402,115 @@
  *   Error in findneedle (haystack.c:384): 'Too much hay'
  *   ~~~
  */
-# define SIR_ERRORFORMAT "Error in %s (%s:%u): '%s'"
+# if !defined(SIR_ERRORFORMAT)
```

Comment:
Are you 100% sure we don't use the size of this buffer hard-coded as a numeric anywhere in an sprintf to avoid truncation warnings?

Reply:
No, so I'll exclude it.